packages/react-magma-dom/src/components/Dropdown/DropdownMenuItem.tsx

```diff
@@ -60,6 +61,21 @@ export function menuBackground(props) {
   return props.theme.colors.neutral200;
 }
 
+function menuItemPadding(props) {
+  //For DropdownExpandableMenu styling with an icon
+  if (props.hasIcon && props.isExpandablePanel) {
+    return `${props.theme.spaceScale.spacing03} ${props.theme.spaceScale.spacing05} ${props.theme.spaceScale.spacing03} 72px`;
+  }
+  //For DropdownExpandableMenu styling without an icon
+  else if (props.isExpandablePanel) {
```

Comment:
You're checking `props.isExpandablePanel` twice, and removed the case for padding when it's not expandable. This needs to be updated as it's breaking other dropdowns: 
![image](https://github.com/cengage/react-magma/assets/91160746/e3e94dd7-945d-4730-b572-acf783d4977a)

Reply:
That actually wasn't removed and still behaves as it did before.

Reply:
Not sure what you mean 🤔

Reply:
Padding remains at `8px 0` when no `DropdownItems` exist as it did before.